### ISSUE TITLE
feature: export settings to file

### ISSUE THREAD

**Issue by jessicajonesunitedkingdom** [emoon/HippoPlayer#111]
Hi

I don't know where you want to store all player's settings when they are made, but I think maybe a good idea is to store them in a XML file or so.
You could then update the player (overwrite executable) and all settings are still in the XML file.

I just hate most players are having default settings when you install an update.
Since the player is cross platform, maybe there is no other choice then a settings file?

I think it's not good to make a QT specific file because the settings should work in another GUI too.

What do you think?

**Comment by emoon** [comment]
Yes that is the plan (by default Qt will store the settings in your home directory so they will persist between different updates) right now now some temporary settings is saved in the root directory (like playlist and song database) but the plan is to not have them there.
